trin-core/src/portalnet/overlay.rs

```diff
@@ -174,6 +174,15 @@ impl<TContentKey: OverlayContentKey + Send, TMetric: Metric + Send>
         self.send_overlay_request(request, direction).await
     }
 
+    /// Process overlay uTP listener event
+    pub fn process_utp_event(&self, event: UtpListenerEvent) {
+        // TODO: Handle overlay uTP events
```

Comment:
Normally I would say to move this into an issue. But if you already have the PR up in draft form, I am fine leaving an in-code todo comment. 👍🏻